package.json

```diff
@@ -36,7 +36,7 @@
     "eslint": "^5.16.0",
     "eslint-plugin-github": "1.0.0",
     "flow-bin": "^0.75.0",
-    "govuk-frontend": "^2.12.0",
+    "govuk-frontend": "^2.13.0",
```

Comment:
I think meant to be here?

Reply:
Following the approach with have in other releases (e.g. govuk_publishing_components), when we release a package we try to update dependencies too